internal/scorecard/alpha/tests/bundle_test.go

```diff
@@ -160,6 +169,44 @@ func TestCRDHaveValidation(t *testing.T) {
 		})
 	}
 }
+func TestStatusDescriptors(t *testing.T) {
+
+	cases := []struct {
+		name       string
+		bundlePath string
+		state      scapiv1alpha2.State
+		function   func(registry.Bundle) scapiv1alpha2.ScorecardTestResult
+	}{
+		{"Should error when CR has format errors", "../testdata/statusdescriptor/error_bundle",
```

Comment:
it is a good practice we add attribute and value instead of just the values. 
E.g 
```go
{ 
name: "value", 
bundlePath: "value"
state: "scapiv1alpha2.State"
function: StatusDescriptorsTest
}
```

It makes easier the understatement.

Reply:
@camilamacedo86 That was causing the `make lint` issues for duplication. Even if we encapsulate, the cases struct would still come up as duplicate, as I have to have two cases structs for `CRDHaveValidation` , and `statusDescriptorTest` functions. So, I am comfortable either the current solution of NOT having Attribute/Value. Or, Go back to Attribute/Value , and  add //nolint:dupl